compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -441,7 +441,7 @@ impl Printer<'tcx> for SymbolMangler<'tcx> {
                     }
                     match sig.abi {
                         Abi::Rust => {}
-                        Abi::C => cx.push("KC"),
+                        Abi::C { .. } => cx.push("KC"),
```

Comment:
Looking through this today, I realize this might actually want to match `Abi::C { unwind: false }`, because `C-unwind` is a distinct ABI? I'm not positive about how these and our other `*-unwind` symbols ought to mangled however.

Reply:
The other `*-unwind` symbols are already mangled as `K*_unwind`

Reply:
Hm, I suspect it's probably not going to matter much in practice, because people would generally use `#[no_mangle]` with `"C unwind"` functions that will be externally linked. It does seem like the intent here is for different ABIs to be differently mangled by default, though.

Reply:
Given `fn foo<T>() {}`, `foo::<extern "C" fn()>` is a different function from `foo::<extern "C-unwind" fn()>`, so they have to be mangled differently.

Reply:
Fixed! :heavy_check_mark: